src/renderer/containers/Dashboard/DashboardContainer.tsx

```diff
@@ -55,13 +60,23 @@ const Dashboard: React.FunctionComponent<IOwnProps &  Pick<IRootState, "auth">>
     };
 
     const onRemoveValidator = (index: number): void => {
-        // delete locally from array
-        const v = [...validators];
-        v.splice(index, 1);
+        setSelectedValidatorIndex(index);
+        setConfirmModal(true);
+    };
+
+    const onConfirmDelete = async (): Promise<void> => {
+        const validatorsData = props.auth.auth;
+        if(validatorsData){
+            const validators =validatorsData.getValidators();
+            const selectedValidatorPublicKey = validators[selectedValidatorIndex].publicKey.toHexString();
+            const selectedV4Keystore = new V4Keystore(
+                path.join(validatorsData.directory,selectedValidatorPublicKey + ".json"));
+            selectedV4Keystore.destroy();
```

Comment:
I think this can throw exception which needs to be displayed as error notification